### ISSUE TITLE
invalid tile number: 80

### ISSUE THREAD

**Issue by timfel** [Wargus/wargus#44]
I get this error if I try to make a new map (I can create and save it, but when I try to load or edit again I get the error) and if I try to edit a campaign map.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2011-03-16T22:14:01Z
 * owner: rocadeier
 * assignee: joris-dauphin
 * the launchpad url was https://bugs.launchpad.net/bugs/736460

**Comment by timfel** [comment]
(by erstmap)
This problem is caused by a tileset not being assigned. In the map file the editor puts:

LoadTileModels("")

This will crash the engine as it expects something like:

LoadTileModels("scripts/tilesets/winter.lua")

You can manually replace LoadTileModels("") with LoadTileModels("scripts/tilesets/winter.lua") in your map file to fix this error.

**Comment by timfel** [comment]
(by pali)
Kyran, can you fix this bug?

**Comment by timfel** [comment]
(by erstmap)
I've tried looking into it, though I haven't had any luck finding where the issue is. The tileset variable is set correctly in the editor because it loads, but it just doesn't seem to make it to the save function.

I don't think I'll be able to fix it any time soon. I haven't looked at it but I've got a hunch that the problem lies within the function that takes care of saving. At the very least that function should assign a default tileset if it doesn't get sent one.
